src/Components/Activitylogcard/ActivityLogCard.tsx

```diff
@@ -12,29 +12,33 @@ import React from 'react';
  */
 
 interface ActivityLogCardProps {
-  title: string;
-  text: string;
-  dateTime: Date;
+  title?: string;
+  text?: string;
+  // dateTime: Date;
   button?: React.ReactNode;
 }
 
-const ActivityLogCard: React.FC<ActivityLogCardProps> = ({ title, text, dateTime, button }) => {
-  const formattedDateTime = format(dateTime, 'dd MMM; hh:mma').toUpperCase();
+const ActivityLogCard: React.FC<ActivityLogCardProps> = ({ 
```

Comment:
main the style have already for this component, and make necessary adjustments like the adding links, if the body is not available let it be static for now